packages/babel-parser/src/types.js

```diff
@@ -1095,6 +1095,7 @@ export type EstreeMethodDefinition = NodeBase & {
 export type EstreeImportExpression = NodeBase & {
   type: "ImportExpression",
   source: Expression,
+  attributes: Expression | null,
```

Comment:
```suggestion
  attributes?: Expression | null,
```
The `attributes` is not assigned when the importAssertion plugin is not enabled.